static/mobile.css

```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-device-width: 750px) {
```

Comment:
768px is probably good.  That is the width of older laptops/netbooks and most tablets, which would all benefit from this layout.